3drepo/3drepo.io
# Issue 4467 - Tabular View fixes

This fixes #4467 

#### Description
Ticket properties disabled outside the viewer now have grey background and a label prefix;
When a user tries to create a ticket that requires the viewer, the "go to viewer" button is enabled;
The edited ticket (selected) now has a grey background to make it clear it is being edited

#### Test cases
Go to the tickets tab and select a template that has required properties that need the viewer:
- try to create a new ticket (button "go to viewer" should be enabled);
- try to edit an existing ticket;
   - the ticket should get a grey background
   - the uneditable properties stand out
